src/Algebra/Graph.hs

```diff
@@ -1086,6 +1191,8 @@ simple op x y
 -- quadratic, i.e. /m = O(m1 * m2)/, but the algebraic representation requires
 -- only /O(m1 + m2)/ operations to list them.
 --
+-- Good consumer of both arguments.
```

Comment:
Does it happen to also be a good producer? I'd hope that `overlays` of `biclique`s can fuse nicely.

Reply:
(Perhaps the list of bicliques does disappear, but not the bicliques themselves...)

Reply:
Indeed, it is a good producer :)